README.md

```diff
@@ -79,11 +79,40 @@ Use the `LangChain` CLI to bootstrap a `LangServe` project quickly.
 
 To use the langchain CLI make sure that you have a recent version of `langchain-cli`
 installed. You can install it with `pip install -U langchain-cli`.
+## Setup
+**Note**: We use `poetry` for dependency management. Please follow poetry [doc](https://python-poetry.org/docs/) to learn more about it.
 
+### 1. Create new app using langchain cli command
 ```sh
-langchain app new ../path/to/directory
+langchain app new my-app
 ```
 
+### 2. Activate `poetry` shell. Every command after step 2 should be executed within `poetry` shell.
+```sh
+poetry shell
+```
+
+### 3. Now, install all the dependencies using `poetry`
+```sh
+poetry install
+```
+
+### 4. Install `langchain-openai` dependency to use ChatOpenAI llm. Do similar for other dependencies you plan to use.
+```sh
+pip intall langchain-openai
```

Comment:
This should be `poetry add langchain-openai`, but we also need to be a bit more vendor agnostic (even though openai is super popular!)

Could we re-write to say something along the lines of:

Set up any relevant 3rd party packages (e.g., `langchain-openai`, `langchain-anthropic`, `langchain-mistral` etc), and set up relevant env variables

Reply:
Done